### ISSUE TITLE
[BUG] massivly slow syncing

### ISSUE THREAD

**Issue by Hardcore-fs** [Chia-Network/chia-blockchain#6707]
1.1.7
there is still an issue with syncing the wallet & network.
it can take 14 hours to  pull in the sync data from just a few days ago.

network is a 1G optical ports are open & external systems can connect
but some nodes are refusing to send data, whilst
![bob22](https://user-images.githubusercontent.com/1309004/121793870-e0b19700-cc35-11eb-9c9a-72221505ac04.jpeg)
 others are sending MB of data

**Comment by Teledyol** [comment]
Using macOS GUI:

Full node is working  on beta external SSD running macOS 15 (monterey), but I wantet to get the main platform working on the internal SSD running macOS 14.5 and sync it up

The Full Node sync is stuck at "Syncing 255,412/426,185" for several hours:

The log for the past ten minutes doesn't really show anything other than that the setup should work once the Status is fully synched:

2021-06-13T09:30:16.235 full_node full_node_server        : WARNING  Banning 117.146.23.226 for 600 seconds
2021-06-13T09:30:16.236 full_node chia.full_node.full_node: ERROR    Error with syncing: <class 'RuntimeError'>Traceback (most recent call last):
  File "chia/full_node/full_node.py", line 640, in _sync
RuntimeError: Weight proof did not arrive in time from peer: 117.146.23.226

2021-06-13T09:30:54.835 full_node chia.full_node.full_node: ERROR    Error with syncing: <class 'RuntimeError'>Traceback (most recent call last):
  File "chia/full_node/full_node.py", line 640, in _sync
RuntimeError: Weight proof did not arrive in time from peer: 171.235.94.145

2021-06-13T09:33:03.722 full_node chia.full_node.full_node: ERROR    Error: Err.UNKNOWN_UNSPENT, Invalid block from peer: {'host': '51.155.123.113', 'port': 8444} 
2021-06-13T09:33:03.725 full_node full_node_server        : WARNING  Banning 51.155.123.113 for 600 seconds
2021-06-13T09:33:07.508 full_node chia.full_node.full_node: ERROR    Error: Err.UNKNOWN_UNSPENT, Invalid block from peer: {'host': '210.152.12.153', 'port': 8444} 
2021-06-13T09:33:07.510 full_node full_node_server        : WARNING  Banning 210.152.12.153 for 600 seconds
2021-06-13T09:33:12.251 full_node chia.full_node.full_node: ERROR    Error: Err.UNKNOWN_UNSPENT, Invalid block from peer: {'host': '36.189.253.253', 'port': 8444} 
2021-06-13T09:33:12.253 full_node full_node_server        : WARNING  Banning 36.189.253.253 for 600 seconds
2021-06-13T09:33:15.809 full_node chia.full_node.full_node: ERROR    Error: Err.UNKNOWN_UNSPENT, Invalid block from peer: {'host': '88.237.86.171', 'port': 8444} 
2021-06-13T09:33:15.811 full_node full_node_server        : WARNING  Banning 88.237.86.171 for 600 seconds
2021-06-13T09:33:19.555 full_node chia.full_node.full_node: ERROR    Error: Err.UNKNOWN_UNSPENT, Invalid block from peer: {'host': '111.193.3.99', 'port': 8444} 
2021-06-13T09:33:19.557 full_node full_node_server        : WARNING  Banning 111.193.3.99 for 600 seconds
2021-06-13T09:33:22.108 full_node chia.full_node.full_node: ERROR    Error: Err.UNKNOWN_UNSPENT, Invalid block from peer: {'host': '156.17.105.235', 'port': 8444} 
2021-06-13T09:33:22.110 full_node full_node_server        : WARNING  Banning 156.17.105.235 for 600 seconds

**Comment by Teledyol** [comment]
Additional Console info:

default	09:28:10.919898-0700	runningboardd	Acquiring assertion targeting [app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] from originator [daemon<com.apple.coreservices.launchservicesd>:121] with description <RBSAssertionDescriptor| "frontmost:654" ID:187-121-804 target:654 attributes:[
	<RBSDomainAttribute| domain:"com.apple.launchservicesd" name:"RoleUserInteractiveFocal" sourceEnvironment:"(null)">
	]>
default	09:28:10.920124-0700	runningboardd	Assertion 187-121-804 (target:[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654]) will be created as active
default	09:28:10.922621-0700	runningboardd	Acquiring assertion targeting [app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] from originator [daemon<com.apple.WindowServer(88)>:153] with description <RBSAssertionDescriptor| "FUSBFrontmostProcess" ID:187-153-805 target:654 attributes:[
	<RBSDomainAttribute| domain:"com.apple.fuseboard" name:"Frontmost" sourceEnvironment:"(null)">,
	<RBSAcquisitionCompletionAttribute| policy:AfterApplication>
	]>
default	09:28:10.922843-0700	runningboardd	Assertion 187-153-805 (target:[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654]) will be created as active
default	09:28:10.922739-0700	runningboardd	Calculated state for app<application.net.chia.blockchain.12918602542.12918602812(501)>: running-active (role: UserInteractiveFocal)
default	09:28:10.926733-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Ignoring jetsam update because this process is not memory-managed
default	09:28:10.927138-0700	runningboardd	Calculated state for app<application.net.chia.blockchain.12918602542.12918602812(501)>: running-active (role: UserInteractiveFocal)
default	09:28:10.927444-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Ignoring suspend because this process is not lifecycle managed
default	09:28:10.928173-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Set darwin role to: UserInteractiveFocal
default	09:28:10.928749-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Ignoring GPU update because this process is not GPU managed
default	09:28:10.929872-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Skipping AppNap state - not lifecycle managed
default	09:28:10.930778-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Ignoring jetsam update because this process is not memory-managed
default	09:28:10.931027-0700	runningboardd	Finished acquiring assertion 187-121-804 (target:[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654])
default	09:28:10.931087-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Ignoring suspend because this process is not lifecycle managed
default	09:28:10.931370-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Ignoring GPU update because this process is not GPU managed
default	09:28:10.931667-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Skipping AppNap state - not lifecycle managed
default	09:28:10.931843-0700	runningboardd	Finished acquiring assertion 187-153-805 (target:[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654])
default	09:28:10.933100-0700	runningboardd	Acquiring assertion targeting [app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] from originator [daemon<com.apple.coreservices.launchservicesd>:121] with description <RBSAssertionDescriptor| "notification:654" ID:187-121-806 target:654 attributes:[
	<RBSDomainAttribute| domain:"com.apple.launchservicesd" name:"LSNotification" sourceEnvironment:"(null)">
	]>
default	09:28:10.933247-0700	runningboardd	Assertion 187-121-806 (target:[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654]) will be created as active
default	09:28:10.933898-0700	runningboardd	Calculated state for app<application.net.chia.blockchain.12918602542.12918602812(501)>: running-active (role: UserInteractiveFocal)
default	09:28:10.933928-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Ignoring jetsam update because this process is not memory-managed
default	09:28:10.933984-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Ignoring suspend because this process is not lifecycle managed
default	09:28:10.934042-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Ignoring GPU update because this process is not GPU managed
default	09:28:10.934146-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Skipping AppNap state - not lifecycle managed
default	09:28:10.934488-0700	runningboardd	Finished acquiring assertion 187-121-806 (target:[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654])
default	09:28:16.928540-0700	runningboardd	Assertion did invalidate due to timeout: 187-121-806 (target:[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654])
default	09:28:17.067679-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Ignoring jetsam update because this process is not memory-managed
default	09:28:17.067715-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Ignoring suspend because this process is not lifecycle managed
default	09:28:17.067753-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Ignoring GPU update because this process is not GPU managed
default	09:28:17.067817-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Skipping AppNap state - not lifecycle managed
default	09:28:17.067795-0700	runningboardd	Calculated state for app<application.net.chia.blockchain.12918602542.12918602812(501)>: running-active (role: UserInteractiveFocal)
default	09:28:27.634344-0700	runningboardd	Invalidating assertion 187-121-804 (target:[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654]) from originator [daemon<com.apple.coreservices.launchservicesd>:121]
default	09:28:27.648793-0700	runningboardd	Invalidating assertion 187-153-805 (target:[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654]) from originator [daemon<com.apple.WindowServer(88)>:153]
default	09:28:27.745644-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Ignoring jetsam update because this process is not memory-managed
default	09:28:27.745647-0700	runningboardd	Calculated state for app<application.net.chia.blockchain.12918602542.12918602812(501)>: running-active (role: UserInteractiveNonFocal)
default	09:28:27.745835-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Ignoring suspend because this process is not lifecycle managed
default	09:28:27.746048-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Set darwin role to: UserInteractiveNonFocal
default	09:28:27.746149-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Ignoring GPU update because this process is not GPU managed
default	09:28:27.746335-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Skipping AppNap state - not lifecycle managed
default	09:31:53.792785-0700	runningboardd	Acquiring assertion targeting [app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] from originator [daemon<com.apple.coreservices.launchservicesd>:121] with description <RBSAssertionDescriptor| "frontmost:654" ID:187-121-836 target:654 attributes:[
	<RBSDomainAttribute| domain:"com.apple.launchservicesd" name:"RoleUserInteractiveFocal" sourceEnvironment:"(null)">
	]>
default	09:31:53.792928-0700	runningboardd	Assertion 187-121-836 (target:[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654]) will be created as active
default	09:31:53.796291-0700	runningboardd	Acquiring assertion targeting [app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] from originator [daemon<com.apple.WindowServer(88)>:153] with description <RBSAssertionDescriptor| "FUSBFrontmostProcess" ID:187-153-837 target:654 attributes:[
	<RBSDomainAttribute| domain:"com.apple.fuseboard" name:"Frontmost" sourceEnvironment:"(null)">,
	<RBSAcquisitionCompletionAttribute| policy:AfterApplication>
	]>
default	09:31:53.796542-0700	runningboardd	Assertion 187-153-837 (target:[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654]) will be created as active
default	09:31:53.794653-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Ignoring jetsam update because this process is not memory-managed
default	09:31:53.793848-0700	runningboardd	Calculated state for app<application.net.chia.blockchain.12918602542.12918602812(501)>: running-active (role: UserInteractiveFocal)
default	09:31:53.798052-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Ignoring suspend because this process is not lifecycle managed
default	09:31:53.799484-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Set darwin role to: UserInteractiveFocal
default	09:31:53.800354-0700	runningboardd	Finished acquiring assertion 187-121-836 (target:[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654])
default	09:31:53.800757-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Ignoring GPU update because this process is not GPU managed
default	09:31:53.799760-0700	runningboardd	Calculated state for app<application.net.chia.blockchain.12918602542.12918602812(501)>: running-active (role: UserInteractiveFocal)
default	09:31:53.802288-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Skipping AppNap state - not lifecycle managed
default	09:31:53.803672-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Ignoring jetsam update because this process is not memory-managed
default	09:31:53.804028-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Ignoring suspend because this process is not lifecycle managed
default	09:31:53.804415-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Ignoring GPU update because this process is not GPU managed
default	09:31:53.805171-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Skipping AppNap state - not lifecycle managed
default	09:31:53.805581-0700	runningboardd	Finished acquiring assertion 187-153-837 (target:[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654])
default	09:31:53.806865-0700	runningboardd	Acquiring assertion targeting [app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] from originator [daemon<com.apple.coreservices.launchservicesd>:121] with description <RBSAssertionDescriptor| "notification:654" ID:187-121-838 target:654 attributes:[
	<RBSDomainAttribute| domain:"com.apple.launchservicesd" name:"LSNotification" sourceEnvironment:"(null)">
	]>
default	09:31:53.807014-0700	runningboardd	Assertion 187-121-838 (target:[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654]) will be created as active
default	09:31:53.808959-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Ignoring jetsam update because this process is not memory-managed
default	09:31:53.809276-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Ignoring suspend because this process is not lifecycle managed
default	09:31:53.809556-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Ignoring GPU update because this process is not GPU managed
default	09:31:53.810162-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Skipping AppNap state - not lifecycle managed
default	09:31:53.810723-0700	runningboardd	Finished acquiring assertion 187-121-838 (target:[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654])
default	09:31:53.810832-0700	runningboardd	Calculated state for app<application.net.chia.blockchain.12918602542.12918602812(501)>: running-active (role: UserInteractiveFocal)
default	09:31:58.309870-0700	WindowServer	Connection added: IOHIDEventSystemConnection uuid:BFE6FAD5-FCAB-43D0-B16E-95600E805EB0 pid:654 process:Chia type:Passive entitlements:0x0 caller:HIToolbox: _TISCopyAttachedKeyboardLanguages + 82 attributes:(null) state:0x0 events:0 mask:0x0
default	09:31:58.311566-0700	WindowServer	Connection removed: IOHIDEventSystemConnection uuid:BFE6FAD5-FCAB-43D0-B16E-95600E805EB0 pid:654 process:Chia type:Passive entitlements:0x0 caller:HIToolbox: _TISCopyAttachedKeyboardLanguages + 82 attributes:(null) state:0x1 events:0 mask:0x0
default	09:31:58.807611-0700	runningboardd	Assertion did invalidate due to timeout: 187-121-838 (target:[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654])
default	09:31:59.009063-0700	runningboardd	Calculated state for app<application.net.chia.blockchain.12918602542.12918602812(501)>: running-active (role: UserInteractiveFocal)
default	09:31:59.009073-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Ignoring jetsam update because this process is not memory-managed
default	09:31:59.009119-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Ignoring suspend because this process is not lifecycle managed
default	09:31:59.009164-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Ignoring GPU update because this process is not GPU managed
default	09:31:59.009402-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Skipping AppNap state - not lifecycle managed
default	09:32:00.590847-0700	runningboardd	Invalidating assertion 187-121-836 (target:[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654]) from originator [daemon<com.apple.coreservices.launchservicesd>:121]
default	09:32:00.605581-0700	runningboardd	Invalidating assertion 187-153-837 (target:[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654]) from originator [daemon<com.apple.WindowServer(88)>:153]
default	09:32:00.710547-0700	runningboardd	Calculated state for app<application.net.chia.blockchain.12918602542.12918602812(501)>: running-active (role: UserInteractiveNonFocal)
default	09:32:00.710653-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Ignoring jetsam update because this process is not memory-managed
default	09:32:00.710988-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Ignoring suspend because this process is not lifecycle managed
default	09:32:00.711271-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Set darwin role to: UserInteractiveNonFocal
default	09:32:00.711522-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Ignoring GPU update because this process is not GPU managed
default	09:32:00.711957-0700	runningboardd	[app<application.net.chia.blockchain.12918602542.12918602812(501)>:654] Skipping AppNap state - not lifecycle managed
default	09:34:58.901242-0700	lsd	Non-fatal error enumerating at <private>, continuing: Error Domain=NSCocoaErrorDomain Code=260 "The file “PlugIns” couldn’t be opened because there is no such file." UserInfo={NSURL=PlugIns/ -- file:///Volumes/com.apple.TimeMachine.localsnapshots/Backups.backupdb/Mac%20Pro/2021-06-13-091647/OWC%20Aura%20Pro%20X2%20-%20Data/Applications/Chia.app/Contents/, NSFilePath=/Volumes/com.apple.TimeMachine.localsnapshots/Backups.backupdb/Mac Pro/2021-06-13-091647/OWC Aura Pro X2 - Data/Applications/Chia.app/Contents/PlugIns, NSUnderlyingError=0x7fed4910ac40 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
default	09:34:58.901336-0700	lsd	- 45683955: Checking whether application is managed at file:///Volumes/com.apple.TimeMachine.localsnapshots/Backups.backupdb/Mac%20Pro/2021-06-13-091647/OWC%20Aura%20Pro%20X2%20-%20Data/Applications/Chia.app//net.chia.blockchain
default	09:34:58.913275-0700	lsd	net.chia.blockchain:5:5:1:1:Building bundle record for app
default	09:34:58.913558-0700	lsd	net.chia.blockchain:5:5:2:1:Built bundle record for app
default	09:34:59.061467-0700	lsd	net.chia.blockchain:5:4:2:1:_LSServerRegisterItemInfo result = 0
default	09:34:59.062119-0700	nehelper	Handling an apps installed notification with bundle IDs (
    "net.chia.blockchain"
)

**Comment by Hardcore-fs** [comment]
> Using macOS GUI:
> 
> Full node is working on beta external SSD running macOS 15 (monterey), but I wantet to get the main platform working on the internal SSD running macOS 14.5 and sync it up
> 
> The Full Node sync is stuck at "Syncing 255,412/426,185" for several hours:
> 
> The log for the past ten minutes doesn't really show anything other than that the setup should work once the Status is fully synched:
> 
> 2021-06-13T09:30:16.235 full_node full_node_server : WARNING Banning 117.146.23.226 for 600 seconds
> 2021-06-13T09:30:16.236 full_node chia.full_node.full_node: ERROR Error with syncing: <class 'RuntimeError'>Traceback (most recent call last):
> File "chia/full_node/full_node.py", line 640, in _sync
> RuntimeError: Weight proof did not arrive in time from peer: 117.146.23.226
> 
> 2021-06-13T09:30:54.835 full_node chia.full_node.full_node: ERROR Error with syncing: <class 'RuntimeError'>Traceback (most recent call last):
> File "chia/full_node/full_node.py", line 640, in _sync
> RuntimeError: Weight proof did not arrive in time from peer: 171.235.94.145
> 
> 2021-06-13T09:33:03.722 full_node chia.full_node.full_node: ERROR Error: Err.UNKNOWN_UNSPENT, Invalid block from peer: {'host': '51.155.123.113', 'port': 8444}
> 2021-06-13T09:33:03.725 full_node full_node_server : WARNING Banning 51.155.123.113 for 600 seconds
> 2021-06-13T09:33:07.508 full_node chia.full_node.full_node: ERROR Error: Err.UNKNOWN_UNSPENT, Invalid block from peer: {'host': '210.152.12.153', 'port': 8444}
> 2021-06-13T09:33:07.510 full_node full_node_server : WARNING Banning 210.152.12.153 for 600 seconds
> 2021-06-13T09:33:12.251 full_node chia.full_node.full_node: ERROR Error: Err.UNKNOWN_UNSPENT, Invalid block from peer: {'host': '36.189.253.253', 'port': 8444}
> 2021-06-13T09:33:12.253 full_node full_node_server : WARNING Banning 36.189.253.253 for 600 seconds
> 2021-06-13T09:33:15.809 full_node chia.full_node.full_node: ERROR Error: Err.UNKNOWN_UNSPENT, Invalid block from peer: {'host': '88.237.86.171', 'port': 8444}
> 2021-06-13T09:33:15.811 full_node full_node_server : WARNING Banning 88.237.86.171 for 600 seconds
> 2021-06-13T09:33:19.555 full_node chia.full_node.full_node: ERROR Error: Err.UNKNOWN_UNSPENT, Invalid block from peer: {'host': '111.193.3.99', 'port': 8444}
> 2021-06-13T09:33:19.557 full_node full_node_server : WARNING Banning 111.193.3.99 for 600 seconds
> 2021-06-13T09:33:22.108 full_node chia.full_node.full_node: ERROR Error: Err.UNKNOWN_UNSPENT, Invalid block from peer: {'host': '156.17.105.235', 'port': 8444}
> 2021-06-13T09:33:22.110 full_node full_node_server : WARNING Banning 156.17.105.235 for 600 seconds


that's "normal...."
it's some guys "spoofing" double spend packets.
they are just warnings.. but the chia code "aborts" the  python thread rather than a graceful exit....


to clear the "blockage"

chia show -a introducer-apne.chia.net:8444
chia show -a introducer-apse.chia.net:8444
chia show -a introducer-or.chia.net:8444 
chia show -a introducer-va.chia.net:8444
chia show -a introducer-eu.chia.net:8444

should add some reference sites...

**Comment by Teledyol** [comment]
> to clear the "blockage"

chia show -a introducer-apne.chia.net:8444
chia show -a introducer-apse.chia.net:8444
chia show -a introducer-or.chia.net:8444
chia show -a introducer-va.chia.net:8444
chia show -a introducer-eu.chia.net:8444

should add some reference sites...

All I got with that was "-bash: chia: command not found

thx

**Comment by Hardcore-fs** [comment]
you need to be in the correct folder & with the python env. set. (. ./activate)
it might be of the form:

venv/bin/chia show -a introducer-eu.chia.net:8444

**Comment by ahnick** [comment]
I'm attempting to sync one of my nodes running on a MacBook Pro up from height 500,000 to 567,210 and it has been more than 24 hours and I'm only halfway there.  Port 8444 open and forwarded.  Most of my connections show 0.0 Down.  The one node that actually shows data being transmitted has 132.7 MiB Down.  It's really unclear to me why this is so slow considering I'm sitting on a 300 Mbps connection.

**Comment by Hardcore-fs** [comment]
you will find that many nodes are rate blocking sync until you get to where the front is.
Seems they don't want to share the mining & traffic, but are quite happy to leach hundreds of MB once you are synced.
and i'm sitting on a multi Gb optical to the isp.

**Comment by ahnick** [comment]
That explanation would make sense to me if I wasn't also running other full nodes that I control that are fully synced to the blockchain.  I'm not rate limiting myself yet I'm only able to transfer very slowly out to my own node.  I've even tried killing other connections on these full nodes and still my rates to my MacBook Pro do not improve.  If I was to just send the blockchain and wallet db files over and sftp connection it would take minutes to complete the transfer.  Instead it takes days for the blockchain to sync up naturally.  I keep thinking there should be some setting that I could adjust that would enable me to increase the bandwidth to my own nodes and rectify this, but I'm yet to come across exactly what that might be.

**Comment by Hardcore-fs** [comment]
it wont work, and it's nothing to do with mac.
even sitting on an optical makes no difference.
I suspect it is a deliberate policy.....

**Comment by ahnick** [comment]
Not sure what you mean. If I have one node running Linux that is fully synced and another node that is not fully synced running MacOS, then I should be able to sync the blockchain between those two nodes as fast as they can transfer data between them.  Now since I first commented I have been able to fully sync the Mac.  What ended up speeding the transfer up was upgrading both nodes to the latest 1.2.1 release before that they were both operating an older version 1.1.6.  In theory, I shouldn't have had to do that, but maybe something was fixed in regards to syncing with the latest release.
